pages/_app.tsx

```diff
@@ -17,11 +18,19 @@ interface ComponentPropType {
   query?: ReturnType<typeof mapRawQueryToState>
 }
 
+const MATOMO_URL = process.env.NEXT_PUBLIC_MATOMO_URL
+const MATOMO_SITE_ID = process.env.NEXT_PUBLIC_MATOMO_SITE_ID
```

Comment:
These new env vars should be added to the `.env.example` file

Reply:
good catch!